.env.test

```diff
@@ -1 +1,2 @@
+DATABASE_URL=mysql://root:@database:3306/wsexport_test
```

Comment:
I _think_ this only applies to Docker, which not everyone uses. Maybe it should go in `.env.test.local` instead, assuming that's a thing?

Reply:
Yup! Done